### PR TITLE
chore: clarify real-time message pricing

### PR DESCRIPTION
**Pull request by kevcodez**
As per https://www.notion.so/supabase/Realtime-pricing-a5ee5b0cadc642b58bc45a1d64565d5d#95b739faec6745ec8370e05fff89c552 and discussion in Slack https://supabase.slack.com/archives/C01B006T9TN/p1673981823087209

### CODE REVIEW

**Review by MildTomato — APPROVED** [review]

File: apps/www/pages/pricing/CHANGELOG.txt [hunk 1]
```diff
@@ -1,5 +1,10 @@
 Pricing Changelog
 
+### 18 Jan 2023
+
+- Clarified over-usage of real-time messages per month
+    - On the PAYG tier, $2.50 per million after incldued 5 million messages
```
Comment (MildTomato) [inline apps/www/pages/pricing/CHANGELOG.txt:6]: "Pro"

**Review by MildTomato — CHANGES_REQUESTED** [review]
misspelling

File: apps/www/pages/pricing/CHANGELOG.txt (hunk 1 above)
Comment (kevcodez) [inline apps/www/pages/pricing/CHANGELOG.txt:6]:
```suggestion
    - On the Pro tier, $2.50 per million after incldued 5 million messages
```

**Review by MildTomato — APPROVED** [review]
🚢

**Pull request merged**